packages/omuse-era5/setup.py

```diff
@@ -12,9 +12,8 @@
 url = 'https://github.com/omuse-geoscience/omuse'
 install_requires = [
     'omuse-framework',
-    'cdsapi'
 ]
-description = 'The Oceanographic Multi-purpose Software Environment - ERA5 interface'
```

Comment:
hi @esclapez,
What's your reason to change this description? I think in general ERA5 is usually written in capitals and this package offers an interface to download ERA5 data from the climate data store. Just "era5" might be slightly more confusing.